cli_ftd/user/sensor/stm32/stm32_i2c_task.h

```diff
@@ -0,0 +1,6 @@
+#ifndef USER_STM32_I2C_SLAVE_H_
+#define USER_STM32_I2C_SLAVE_H_
+
+extern void stm_i2c_comm_taskCreate(void);
```

Comment:
Why is this extern? You are defining the function in stm32_i2c_task.c right?

Reply:
extern functions are only required for a very specific use case. We are not creating external user defined modules. Everything is being written and consumed by us.

Reply:
Done. Updated in new PR.